### ISSUE TITLE
[FEATURE] Add message persistance to event_rabbitmq

### ISSUE THREAD

**Issue by gtanetwork** [OpenSIPS/opensips#2252]
Hi, 

It appears that RabbitMQ message persistence( (rabbitmq delivery_mode:2)  is disabled by default on 3.1.x/event_rabbitmq module (however enabled on basic rabbitmq). 

This fact results in a potential loss of CDR data, if RabbitMQ server crashes or needs to be restarted for any reason while there are messages sitting in the queue. 
It would be great to make the persistence configuration on this module, hence no CDR loss is possible. 

Thank you in advance.

**Comment by razvancrainea** [comment]
I agree, it would be nice to have a way of specifying a set of properties per connection, perhaps part of the URL of the socket.
Re-opening this and flagging it as a feature request.

**Comment by rvlad-patrascu** [comment]
I did something similar to what @razvancrainea is suggesting for specifying the TLS domain to use for a specific connection. We can add an option in the event socket for persistence as well.
